exercises/raindrops/uRaindropsTest.pas

```diff
@@ -0,0 +1,179 @@
+unit uRaindropsTest;
+
+interface
+uses
+  DUnitX.TestFramework;
+
+type
+
+  [TestFixture]
+  RaindropsTest = class(TObject)
+  public
+    [Test]
+    procedure the_sound_for_1_is_1;
+
+    [Test]
+    [Ignore('Comment these lines to run these tests')]
```

Comment:
My intention is for the comment to be generic and infer that all `[Ignore]` flags should be commented in order to activate a particular test.  I see in other tracks that they just include the comment for all ignores.  Also I am not seeing any ignore included on first tests.  I'm just trying to emulate what I am seeing on other tracks.